### PR TITLE
check job-board response status code on image selection

### PR DESCRIPTION
**Pull request by igorwwwwwwwwwwwwwwwwwwww**

### CODE REVIEW

File: image/api_selector.go [hunk 1]
```diff
@@ -102,16 +103,27 @@ func (as *APISelector) makeImageRequest(urlString string, bodyLines []string) (*
...
 		defer resp.Body.Close()
+
 		responseBody, err = ioutil.ReadAll(resp.Body)
-		return
+		if err != nil {
+			return err
+		}
+
+		if resp.StatusCode != 200 {
+			return errors.Errorf("expected 200 status code, received status=%d body=%q",
```
Comment (henrikhodne) [inline image/api_selector.go]: Would it be a good idea to include from where we expected the 200 status code? Something like "expected 200 status code from Job Board, received…".

Comment (emdantrim) [inline image/api_selector.go]: don't want to be pedantic, but why not `job-board` instead of `Job Board`?

**Comment by igorwwwwwwwwwwwwwwwwwwww** [comment]
If job-board has any kind of intermittent error that causes it to 500 we would probably want to retry. e.g. what if the dyno crashes or such.

File: image/api_selector.go (hunk 1 above)
Comment (igorwwwwwwwwwwwwwwwwwwww) [inline image/api_selector.go]: updated!

**Comment by igorwwwwwwwwwwwwwwwwwwww** [comment]
PR is now passing! 🎈

**Pull request merged**